man: create xdg-ninja.1 man page

Ballasi opened the pull request:
Because man pages are cool.

You can preview it by using `$ man man/xdg-ninja.1`

I can also edit the man page if you think it might require some changes before pushing. I was wondering about adding a paragraph for the optional dependencies (glow and alternatives) but figured it would be a better fit to have that in the `README.md` as a one-time installation information.

b3nj5m1n commented:
Very cool, thank you!

It seems there's a small typo, the options are named as `strip-ok`, `strip-unsupported`, etc., when they're actually named `skip-ok`, `skip-unsupported`.

> I was wondering about adding a paragraph for the optional dependencies

I think there should at least be a paragraph about the recommended dependencies, so `jq` and `glow`, although I'm not really familiar with the best practices for man pages.

Apart from that it looks good!

Ballasi commented:
> It seems there's a small typo, the options are named as `strip-ok`, `strip-unsupported`, etc., when they're actually named `skip-ok`, `skip-unsupported`.

You are right! I actually misread `strip`, that's my bad. Fixed.

> > I was wondering about adding a paragraph for the optional dependencies
> 
> I think there should at least be a paragraph about the recommended dependencies, so `jq` and `glow`, although I'm not really familiar with the best practices for man pages.

Since `jq` is a required (and not a recommended) dependency and the script fails when running without it, I would tend to not have that in the `man` page, because of the way I see man pages ; documentation on how to use a program and not how to install it.

I think it would make sense to add a note about optional dependencies as the user can have a different behavior depending on that during runtime.

For now I have just added a note about `glow`, if you prefer also adding a note about `jq` too I can squash that in.

b3nj5m1n commented:
Looks good to me. Thanks again for all your help with this project!
